recipe/shopware6.php

```diff
@@ -88,9 +90,37 @@
         if ($installed === 'No' || $active === 'No') {
             run("cd {{release_path}} && bin/console plugin:install --activate $plugin");
         }
+    }
+});
+task('sw:plugin:migrate:all', static function(){
+    $plugins = explode("\n", run('cd {{release_path}} && bin/console plugin:list'));
```

Comment:
We should extract the listing of the plugins and maybe even make a json out of it, so we can reuse it later?

Reply:
Yeah I thought of that, but it needs to update in the meantime so we now which plugins are actually installed & active.